backend/src/test/java/gov/cdc/usds/simplereport/test_util/JsonTestUtils.java

```diff
@@ -75,7 +77,8 @@ public static void assertJsonNodesEqual(
               fieldPath + "[" + i + "]");
         }
       } else { // Otherwise assert values are equal
-        assertThat(actualFieldValue.toString()).isEqualTo(expectedFieldValue.toString());
+        log.debug("Comparing " + fieldPath);
```

Comment:
do we need this?

Reply:
Useful when debugging the expected and actual JSON output for the FHIR bundler, but not necessary to keep in so I could comment out or remove